### ISSUE TITLE
Background Alpha value behavior is not the same in Editor and Player

### ISSUE THREAD

**Issue by PatrickA-RiseVision** [Rise-Vision/rise-vision-apps#1079]
### What is the expected behavior?
If the Alpha (transparency) value of a background is not set, Editor and Player behave the same way.

### What is the actual behavior?
In Editor and Preview, the default value of Alpha appears to be 1, and background colors are visible.  In Player, the default value of Alpha appears to be 0, and background colors are not visible.

### What steps can be taken to replicate the problem?
1. Create a blank presentation and set a background color.
2. Remove the 4th value in the text input for the background color.  It should look like this: rgba(0,0,0)
3. Preview the presentation.
4. Run the presentation on your Display.
5. Note that if you used the values above, Preview will be black and Player will be white.

**Comment by alex-deaconu** [comment]
@PatrickA-RiseVision thanks for adding this issue. It is an interesting one.

After testing in Electron, Chrome OS and Local Chrome, it seems this issue is specific to the Electron Player. However, the cause is that the Electron Player is on Chrome version 65. I tested with a dev machine running Chrome 69 on Electron and the issue no longer occurs.

The issue manifests itself when trying to configure an RGBA color with the `A` (alpha) value missing. For example `rgba(0,0,0)`. When that HTML is rendered, the `backgroundColor` property it is actually removed. When trying to add it back in the Dev tools show an `"Invalid property value"` error. While it seems the color is incorrectly formatted, I'm pretty sure the standard allows for the missing alpha value. From my experience with other Chrome versions this worked as expected.

Basically, while we could add validation for `rgba(#,#,#)` values or automatically convert to `rgb(#,#,#)`, it's probably best to simplify and wait for an Electron version update which is in the works.

@tejohnso @andrecardoso @alanclayton @ezequielc FYI. My recommendation is we park and re-test once Electron has been updated.

Thanks!

**Comment by alanclayton** [comment]
@alex-deaconu Thanks and agreed re: fixing with electron upgrade. @PatrickA-RiseVision let us know if that makes sense to you. @andrecardoso will be pursuing this in the coming weeks.

**Comment by alex-deaconu** [comment]
To confirm, the workaround for this is to set a `1` for the `rgba(#,#,#)` value to `rgba(#,#,#,1)`. This should fix the issue on the Display. Thanks.

**Comment by PatrickA-RiseVision** [comment]
@alanclayton @alex-deaconu that makes sense.  We can close this.

**Comment by alex-deaconu** [comment]
Thanks @alanclayton @PatrickA-RiseVision, closing!
